File: CppCompiler/nestedforloop8.cpp
#include<iostream>
using namespace std;

int main()
{
int i,n,j;
double sum=0,sum1=0;
cout<<"enter no";
cin>>n;
for(i=1;i<=n;i++)
{
for(j=1;j<=i;j++)
{
cout<<j;
sum=sum+j;
if(j<i)
{
cout<<"+";
}
}
cout<<"/"<<i;
if(i<n)
{
cout<<" + ";
}
sum1=sum1+(double)sum/i;
sum=0;
}
cout<<"sum of seriese"<<sum1;
}

File: python programming/nestedforloop8.py
n=int(input('enter no of term:-'))
i=1
sum=0
ans=0
k=1
term=1
while term<=n:
	f=0
	for j in range(2,int(k/2+1)):
		if k%j==0:
			f=1
	if f==0:
		for j in range(1,i+1):
			print(j,end='')
			if j<i:
				print('+',end='')
			sum+=j
		print(f'/{k}',end='')
		ans=ans+sum/k
		sum=0
		if term<n:
			print(' + ',end='')
		i+=1
		term+=1
	k+=1
		
print('\b\nsum of seriese=',ans)
		
			
	
	
		
	